quilt/tools/const.py

```diff
@@ -10,6 +10,7 @@ class TargetType(Enum):
     PANDAS = 'pandas'
     FILE = 'file'
 
+SERVER_TIME_F = "%Y-%m-%dT%H:%M:%S"
```

Comment:
Because this acts as a coordination point between client and server, it should go in core.py instead of const.py (so eventually the server could use it to guarantee that it delivers dates in the expected format).